src/Identity/Core/src/DTO/IdentityEndpointsJsonSerializerContext.cs

```diff
@@ -7,6 +7,13 @@ namespace Microsoft.AspNetCore.Identity.DTO;
 
 [JsonSerializable(typeof(RegisterRequest))]
 [JsonSerializable(typeof(LoginRequest))]
+[JsonSerializable(typeof(RefreshRequest))]
+[JsonSerializable(typeof(ResetPasswordRequest))]
+[JsonSerializable(typeof(ResendEmailRequest))]
+[JsonSerializable(typeof(InfoRequest))]
+[JsonSerializable(typeof(InfoResponse))]
+[JsonSerializable(typeof(TwoFactorRequest))]
+[JsonSerializable(typeof(TwoFactorResponse))]
```

Comment:
Would any of these classes benefit from being structs? I think minimal tries to avoid boxing when it can, and Json avoids boxing sometimes too.

Reply:
These are all immutable, so structs are certainly an option. Given that all of these are used for requests that involve some sort of database interaction in the normal case, I don't think it will make a huge difference. It will matter more once we make these public (which I hope to do soon).